certification/internal/policy/container/has_modified_files.go

```diff
@@ -43,7 +43,7 @@ func (p *HasModifiedFilesCheck) getDataToValidate(ctx context.Context, imgRef ce
 	// rpm, dnf, yum, etc. being installed in the image.
 	pkgList, err := rpm.GetPackageList(ctx, imgRef.ImageFSPath)
 	if err != nil {
-		return nil, err
+		return nil, errors.ErrRPMPackageList
```

Comment:
Can the preflight error wrap `err` here?

Reply:
I choose not to wrap this error here and in `has_prohibited_packages.go` to reduce the noise, but maybe for those two cases it makes sense to add wrapping.